test/version_commands.bats

```diff
@@ -238,6 +238,15 @@ teardown() {
   unset ASDF_DUMMY_VERSION
 }
 
+@test "shell wrapper function with --unset should unset ENV var" {
+  source $(dirname "$BATS_TEST_DIRNAME")/asdf.sh
+  asdf shell "dummy" "1.1.0"
+  [ $(echo $ASDF_DUMMY_VERSION) = "1.1.0" ]
+  asdf shell "dummy" "--unset"
```

Comment:
Just curious, why the quotes around these arguments? Flags are typically not quoted.

Reply:
Ah! I've copied and edited the line above:) I'll remove quotes from `--unset`

Reply:
done in bc309c0